__tests__/app.test.js

```diff
@@ -154,4 +154,53 @@ describe('GET comments', () => {
                 expect(comments).toEqual([])
             })
     })
+})
+describe('POST 201 create comment for article', () => {
+    test('creates new comment for passed article_id and responds with that comment', () => {
+        const newComment = { username: "rogersop", body: "This is a test comment" }
+        const insertedComment = { author: "rogersop", body: "This is a test comment", votes: 0, article_id: 10 }
+        return request(app)
+            .post('/api/articles/10/comments').send(newComment)
+            .expect(201)
+            .then(({ body }) => {
+                const { comment } = body
+                expect(comment).toMatchObject(insertedComment)
+            })
+    })
```

Comment:
I'd add in another test for a 201: Ignores any unnecessary properties on the request body.